Packages/com.unity.inputsystem/InputSystem/Utilities/InputStatistics.cs

```diff
@@ -0,0 +1,115 @@
+using Unity.Profiling;
+
+namespace UnityEngine.InputSystem
+{
+    /// <summary>
+    /// Input Statistics for Unity Profiler integration.
+    /// </summary>
+    internal static class InputStatistics
```

Comment:
I'd create a `Editor/Internal/Profiler` folder and but everything in there. I find odd to have this class next to a bunch of "Utilities" classes.